src/components/Tabs.jsx

```diff
@@ -7,28 +7,48 @@ import classNames from 'classnames'
 class Tabs extends React.Component {
     constructor(props, context) {
         super(props, context)
-        this.state = { activeTab: props.defaultTab || 0 }
+
+        const { defaultTab, onChange } = props
+        const children = React.Children.toArray(this.props.children)
+
+        if (defaultTab || onChange) {
```

Comment:
We should add something like
```
const isDefaultTabDefined = defaultTab || defaultTab === 0
```
and use that in the if-conditions otherwise it will result in unwanted behaviour

Reply:
Good catch 👍 just fixed it.